### PR TITLE
Part2

### PR DESCRIPTION
**Pull request by rousell**
@hagansmith I opened this PR because it looks like you are done with Part2. please confirm and i'll give this a final review.

### CODE REVIEW

**Comment by hagansmith** [comment]
I am an idiot!. Yes I am done.

**Comment by rousell** [comment]
Okay cool! This looks pretty good - the only thing that I'd like you to change is some of the values in your json file. For values that are pure numbers, make the values numbers not strings. For the values that are booleans, make those values either the keyword true or false (again, don't make them into strings).

Ping me when you've fixed!

**Comment by rousell** [comment]
@hagansmith looks good! merging in

File: db/contacts.json [hunk 1]
```diff
@@ -0,0 +1,134 @@
+{
+  "contacts": [
+    {
```
Comment (morecallan) [inline db/contacts.json:3]: @hagansmith - this should be an object of objects with keys for each contact. Annnd, while you are in here, you may as well add uid to contacts, right? 💁

**Review by morecallan — COMMENTED** [review]
@hagansmith Reopening so you can wrap up the json file.

**Comment by morecallan** [comment]
@hagansmith - JK - you'll have to reopen it but just reopen, make the changes here, and we'll merge it back in!

**Pull request merged**